tests/request/test_support_pages.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from consultation_analyser.factories import ConsultationFactory
+
+
+@pytest.mark.django_db
+def test_no_login_support_pages(client):
+    ConsultationFactory(slug="consultation-slug")
+    support_urls = [
```

Comment:
This is ok for now but I wonder if we could prove this in a simpler/faster way — will have a think!

Reply:
Def be keen to improve this. Did try and write something more generic to go through all the URLs in support pages - but got stuck at the fact that some of them have parameters (and didn't want to spend too much time just on this test).

Reply:
Great. I wonder if we could wrap the views in a class and put the decorator in that, somehow? anyway, not for this PR!